core/src/main/scala/stryker4s/report/JsonScoreReporter.scala

```diff
@@ -0,0 +1,55 @@
+package stryker4s.report
+
+import grizzled.slf4j.Logging
+import stryker4s.config.Config
+import stryker4s.files.FileIO
+import stryker4s.model.MutantRunResults
+import stryker4s.report.mapper.MutantRunResultMapper
+
+/**
+  * Ideally this will become a direct reporting to dashboard,
+  * https://dashboard.stryker-mutator.io/api/reports,
+  * but it will be hard right now to get the apiKey,
+  * the branch name and the repositorySlug from within the code.
+  * The envision solution is to print out this simple json file that can be parse with `jq`
+  * and then use curl from within Travis or similar system.
+  * {{{
+  *   curl -X POST \
+  *   https://dashboard.stryker-mutator.io/api/reports \
+  *   -H 'Content-Type: application/json' \
+  *   -d '{
+  *     "apiKey": "[API_KEY]",
+  *     "repositorySlug": "github.com/[USER]/[REPO_NAME]",
+  *     "branch": "[BRANCH]",
+  *     "mutationScore": [MUTATION_SCORE]
+  *   }'
+  * }}}
+  */
+class JsonScoreReporter(fileIO: FileIO)(implicit config: Config)
+    extends FinishedRunReporter
+    with MutantRunResultMapper
+    with Logging {
+
+  private case class ScoreResult(
+      mutationScore: Double,
+      durationSeconds: Long
+  ) {
+
+    def toJson: String = {
+      import io.circe.generic.auto._
+      import io.circe.syntax._
+      this.asJson.noSpaces
+    }
+  }
+
+  override def reportRunFinished(runResults: MutantRunResults): Unit = {
+    val targetLocation = config.baseDir / s"target/stryker4s-result-${System.currentTimeMillis()}"
```

Comment:
Keep in mind this means the html and JSON are written to different folders, as they will be called after each other (different currentTimeMillis)

Reply:
Fixed in the coming push